stackanetes/parameters.yaml

```diff
@@ -54,11 +54,18 @@ ceph:
   glance_user: "glance"
   glance_pool: "images"
   glance_keyring: ""
+  rgw_user: "ironic"
+  admin_keyring: ""
   nova_pool: "vms"
   # Random uuid used for Nova / Cinder communication.
   # Should not be modified after the initial deployment.
   secret_uuid: bbc5b4d5-6fca-407d-807d-06a4f4a7bccb
 
+rgw:
```

Comment:
Let's not expose this block in the main configuration file if configuring these values isn't really that useful for the end-user - to avoid creating confusion and overhead. It looks more like internal parameters to me? Instead, we should be sure to have sensitive/matching defaults in both packages that use them.